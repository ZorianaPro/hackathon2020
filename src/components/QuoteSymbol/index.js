import React from 'react';

const QuoteSymbol = (props) => (
  <symbol
    viewBox="0 0 24 24"
    className="QuoteSymbol"
    id="QuoteSymbol"
    xmlns="http://www.w3.org/2000/svg"
    {...props}
  >
    <path d="M0 0h24v24H0z" fill="none"/>
    <path fill="#ff406a" d="M6 17h3l2-4V7H5v6h3zm8 0h3l2-4V7h-6v6h3z"/>
  </symbol>
);
export default QuoteSymbol;
