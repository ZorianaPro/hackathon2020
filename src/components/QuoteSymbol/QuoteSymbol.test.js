import React from 'react';
import { shallow } from 'enzyme';
import QuoteSymbol from './';

describe('QuoteSymbol', () => {
  it('renders without crashing', () => {
    expect(shallow.bind(shallow, <QuoteSymbol />)).not.toThrow();
  });

  it('renders as expected', () => {
    expect(shallow(<QuoteSymbol />)).toMatchSnapshot();
  });
});
