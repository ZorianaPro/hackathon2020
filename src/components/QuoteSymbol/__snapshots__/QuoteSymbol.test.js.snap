// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`QuoteSymbol renders as expected 1`] = `
<symbol
  className="QuoteSymbol"
  id="QuoteSymbol"
  viewBox="0 0 24 24"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M0 0h24v24H0z"
    fill="none"
  />
  <path
    d="M6 17h3l2-4V7H5v6h3zm8 0h3l2-4V7h-6v6h3z"
    fill="#ff406a"
  />
</symbol>
`;
